src/app/beer_garden/api/http/handlers/authorization_handler.py

```diff
@@ -15,19 +16,29 @@
     RequestForbidden,
 )
 from beer_garden.authorization import (
-    user_has_permission_for_object,
-    user_permitted_objects,
-    user_permitted_objects_filter,
+    ModelFilter,
+    QueryFilterBuilder,
+    check_global_roles,
 )
-from beer_garden.db.mongo.models import User
+
+# from beer_garden.db.mongo.models import User
 from beer_garden.errors import ExpiredTokenException, InvalidTokenException
 
 
 class AuthorizationHandler(BaseHandler):
     """Handler that builds on BaseHandler and adds support for authorizing requests
     via a jwt access token supplied in the Authorization header"""
 
-    def get_current_user(self) -> "User":
+    GARDEN_ADMIN = "GARDEN_ADMIN"
```

Comment:
Permissions types were defined in brewtils class Role and redefined here. It would be nice if there was a way to link the constants somehow or is that undesirable because versions of each may differ?

Reply:
Should we do this like the `Events` model where we can reference things like `brewtils.models.PermissionLevel.GARDEN_ADMIN` (or just `PermissionLevel.GARDEN_ADMIN`)